CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -10,7 +10,7 @@ <h2 class="definition-title definition-title--search-result">
 
         <dfn class="definition-title__word" data-cy="definition-title">
           {% if result.is_lemma %}
-          <a class="definition-title__link" href="{{ result.lemma_wordform.lemma_url }}">
+          <a class="definition-title__link" data-cy="lemma-link" href="{{ result.lemma_wordform.lemma_url }}">
```

Comment:
for better integration testing. `[data-cy="definition-title"]` definied above gives both (unclickable) matched wordforms and (clickable) lemmas.

while `[data-cy="lemma-link]"`should be 100% a link and directs to the paradigm page